code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/ShumihinSergeyTest.java

```diff
@@ -0,0 +1,36 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+import static org.junit.Assert.*;
+
+public class ShumihinSergeyTest {
+    @Test
+    public void canAddComplexNumbers() {
+        ComplexNumber firstNumber = new ComplexNumber(1.0, 0.0);
+        ComplexNumber secondNumber = new ComplexNumber(0.0, -1.0);
+
+        ComplexNumber resultNumber = firstNumber.add(secondNumber);
+
+        assertEquals(new ComplexNumber(1.0, -1.0), resultNumber);
+    }
+
+    @Test
+    public void canInitFromString() {
+        ComplexNumber firstNumber = new ComplexNumber("1.0", "0.0");
+        ComplexNumber secondNumber = new ComplexNumber(0.0, -1.0);
```

Comment:
I don't see any need in creating second number and addition of this number in test called `canInitFromString`. All logic should be restricted to init and assert, I suppose